IBPSA/Fluid/Movers/BaseClasses/IdealSource.mo

```diff
@@ -4,8 +4,11 @@ model IdealSource
   extends IBPSA.Fluid.Interfaces.PartialTwoPortTransport(show_T=false);
 
   // Quantity to control
-  parameter Boolean control_m_flow "= false to control dp instead of m_flow"
+  parameter Boolean control_m_flow "= false to m_flow"
     annotation(Evaluate = true);
+  parameter Boolean control_dp = not control_m_flow "= false to control dp"
```

Comment:
@Mathadon I think the comments for `control_m_flow` and `control_dp` are not easy to understand for the user. I would expect that `control_dp` should be `true` to control `dp`, not `false`. And `"= false to m_flow"` could use a verb.